modules/claims_api/lib/evss_service/base.rb

```diff
@@ -18,7 +18,11 @@ def initialize(request = nil)
       def submit(claim, data)
         @auth_headers = claim.auth_headers
 
-        client.post('submit', data).body
+        begin
+          client.post('submit', data).body
+        rescue => e
+          ClaimsApi::Logger.log('526', detail: "EVSS CONTAINER submit error: #{e.original_body}")
```

Comment:
hmm... this throws a 500 to the consumer, right? I know ultimately we won't see this as we will have these calls in a job, so the consumer would never know, but I wonder if we find a way to bypass this until we introduce the entirety of the flow. Just thinking that we may hinder our consumers developing against this... granted they know it's actively being modified.